extensions/functions_aggregate_decimal.yaml

```diff
@@ -0,0 +1,41 @@
+%YAML 1.2
+---
+aggregate_functions:
```

Comment:
the name of this file is weird. in other cases type names in filenames are about input types and this one is about output types. I suggest we rename file to something that is more descriptive. Maybe also add a little text to the functions clarifying how they are different than classic count functions.